decidim-meetings/app/views/decidim/meetings/meetings/_meeting_agenda.html.erb

```diff
@@ -1,31 +1,33 @@
-<div class="section agenda-section">
-  <div class="flex--sbe">
-    <h2 class="section-heading"><%= translated_attribute(meeting.agenda.title) %></h2>
+<div data-meeting-agenda>
+  <div class="meeting__agenda-title">
+    <%= icon "list-ordered", class: "w-6 h-6 text-tertiary fill-current" %>
+    <h3 class="h4"><%= translated_attribute(meeting.agenda.title) %></h3>
```

Comment:
Currently wrong heading level but this is correct after the meeting drawer's title is changed to `<h2>`.